test/language/expressions/class/elements/field-definition-accessor-no-line-terminator.js

```diff
@@ -0,0 +1,31 @@
+// This file was procedurally generated from the following sources:
+// - src/class-elements/field-definition-accessor-no-line-terminator.case
+// - src/class-elements/default/cls-expr.template
+/*---
+description: Valid accessor FieldDefinition, ClassElementName, PropertyName Syntax (field definitions in a class expression)
+esid: prod-FieldDefinition
+features: [decorators, class]
+flags: [generated]
+info: |
+    FieldDefinition[Yield, Await] :
+      accessor [no LineTerminator here] ClassElementName[?Yield, ?Await] Initializer[+In, ?Yield, ?Await]opt
+
+---*/
+
+
+var C = class {
+  accessor
+  $;
+  static accessor
+  $;
+
+}
+
+let c = new C();
+
+assert.sameValue(Object.hasOwnProperty.call(C.prototype, 'accessor'), false);
+assert.sameValue(Object.hasOwnProperty.call(C.prototype, '$'), false);
+assert.sameValue(Object.hasOwnProperty.call(C, 'accessor'), true);
+assert.sameValue(Object.hasOwnProperty.call(C, '$'), true);
```

Comment:
Class `C` is basically `class { accessor; $; static accessor; $; }` due to automatic semicolon insertion (and the fact `FieldDefinition` of an accessor cannot contain `LineTerminator`). So, there is not static `$` field. In other words, this line should expect `false` (not `true`).